content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,190 @@
+﻿---
+title: April '21 Heartbeat
+date: 2021-04-16
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires, and more!
+descriptionLong: |
+  Monthly updates are here! Read all about new lessons and a tutorials from
```

Comment:
btw, just a tip - this section supports links, and _different_ **text** `styles`